src/test/kotlin/calculator/CalculatorTest.kt

```diff
@@ -0,0 +1,52 @@
+package calculator
+
+import org.assertj.core.api.Assertions.assertThat
+import org.assertj.core.api.Assertions.assertThatRuntimeException
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.CsvSource
+import org.junit.jupiter.params.provider.NullAndEmptySource
+import org.junit.jupiter.params.provider.ValueSource
+
+class CalculatorTest {
+
+    @ParameterizedTest
+    @CsvSource("0,1,2|3", "1,2,3|6", delimiter = '|')
+    fun `기본 구분자와 숫자 계산 성공`(target: String, expected: Int) {
+        assertThat(SUT.calculate(target)).isEqualTo(expected)
+    }
+
+    @ParameterizedTest
+    @CsvSource("""//;\n0;2;3|5""", """//;\n1;0;3|4""", delimiter = '|')
```

Comment:
```suggestion
    @CsvSource("""//.\n0.2.3|5""", """//.\n1.0.3|4""", delimiter = '|')
```
커스텀 구분자가 마침표라면 테스트 결과가 어떻게 될까요?

Reply:
정말 생각도 못했네요...!! 감사합니다 :)